src/components/bootstrapExecutablesService.ts

```diff
@@ -65,12 +65,16 @@ export class BootstrapExecutablesService {
 		return executableUri;
 	}
 
-	private async __bootstrapCodemodEngineRustExecutableUri(): Promise<Uri> {
+	private async __bootstrapCodemodEngineRustExecutableUri(): Promise<Uri | null> {
 		const platform =
 			process.platform === 'darwin'
 				? 'macos'
 				: encodeURIComponent(process.platform);
 
+		if (platform === 'win32') {
+			return null;
```

Comment:
I think we could compile the Rust app for Windows as well, I just couldn't cross compile it with Linux.

Reply:
but not in current release, right?

Reply:
no, not yet :)